### DISCUSSION TITLE
Alternative to Xiaomi Wall Plate Switch WXKG06LM/WXKG02LM supporting binding

### DISCUSSION THREAD

**Discussion by Morphy99** [Koenkk/zigbee2mqtt#5625] [Devices]
First off, does anyone know what the difference is between WXKG06LM/WXKG02LM?

On the official Aqara website WXKG03LM only is listed which is the double switch version of WXKG02LM and looks like this:
![image](https://user-images.githubusercontent.com/5853979/104011828-a9cae700-51a6-11eb-95c2-0e7651468c6e.png)

The WXKG06LM appears to have lights on the front and be slightly thinner so maybe a newer version:
![image](https://user-images.githubusercontent.com/5853979/104011877-bbac8a00-51a6-11eb-8157-e6ee03faa2b1.png)

Second, does anyone know of an alternative switch that has the same form factor (replacing existing manual switch) but supports binding?

**Comment by d-c-u-p** [comment]
The WXKG06LM is supposed to send a "release" action. The other one not.

Could somebody confirm that?
